server/routes/index.test.ts

```diff
@@ -65,6 +65,13 @@ describe('check response headers are set correctly', () => {
         "font-src 'self'",
         "img-src 'self' https://www.google-analytics.com",
         "connect-src 'self' https://www.google-analytics.com",
+        "base-uri 'self'",
```

Comment:
@NeilLuptonMoJ Just wanted to understand what is the significance of this change while upgrading helmet to a new version?

Reply:
these are the default options set by helmet v5, which changes the `contentSecurityPolicy` headers that are set.  I've tested locally with these settings, all working and with no errors being shown in console.

"helps mitigate cross-site scripting attacks, among other things."
see https://github.com/helmetjs/helmet#reference section `helmet.contentSecurityPolicy(options)`